desktop_switcher.ahk

```diff
@@ -76,6 +76,13 @@ getSessionId()
 switchDesktopByNumber(targetDesktop)
 {
     global CurrentDesktop, DesktopCount
+    
+    ; There are three issues with switching desktops with active windows in intermediary desktops:
+    ; 1. Occasionally, not all "go right" or "go left" hotkeys are resulting in a switched desktop, this results in switcher getting stuck midway (not at the destination desktop, while at the end CurrentDesktop gets itself set to the target desktop number)
+    ; 2. Switching is not instantaneous anymore, this introduces rapid flashing (each desktop shows itself for a brief moment)
+    ; 3. Flashing orange notifications on taskbar on intermediary windows (https://github.com/pmb6tz/windows-desktop-switcher/issues/8)
+    ; Therefore we will activate taskbar
+    WinActivate, ahk_class Shell_TrayWnd
```

Comment:
Can you explain this change a bit? I don't quite understand how this alone improves performance?

Reply:
Yes, of course!

Let's assume I have 10 different desktops and want to switch from 2 to 9. 
- When there are little or no windows open in desktops [2; 8] it switches instantaneously, as if I were switching to desktop 3. Awesome! 
- However, if I do have some active windows in intermediate windows, the  ctrl+win+right gets sent to the active windows first and only **after**  that to the Windows itself. This results in a short but noticeable delay between switching each desktop from 2 to 9 (each desktop flashes for a brief moment).

This line of code solved the delays by switching focus from the current active window to the taskbar. It also solves two other bugs as a side effect - orange flashing taskbar buttons after reaching destination desktop, and sometimes the switcher getting stuck in a desktop before reaching target desktop. 

In my own setup I also reactivate the active window in the current desktop, so that this line of fix is completely unnoticeable

Reply:
Sounds good--I have yet to test this yet but it does sound like a reasonable approach. Out of curiosity, why not include the fix mentioned in your last sentence to make this completely unobtrusive?

Reply:
@Elijas I appreciate the thorough breakdown; very helpful :-). I'll accept this without the focus fix for now, as I do think it's an improvement that addresses an existing bug. Thank you!

Reply:
No problem!:)

Although, I've left one bug - if user attempts to switch to left in Desktop 1 (or right at last desktop) then the current window will be unfocused

Fixed it in #20